Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Use phusion/baseimage if problems arise
+FROM ubuntu:18.04
+MAINTAINER Collin Avidano collin.avidano@gmail.com
+
+# Setup apt to be happy with no console input
+ENV DEBIAN_FRONTEND noninteractive
+
+# setup apt tools and other goodies we want
+# TODO Double check these later for which are really necessary
+RUN apt-get update --fix-missing && apt-get -y install udev git nano vim software-properties-common sudo apt-utils wget curl htop iputils-ping less bsdmainutils scons screen && apt-get clean
```

Comment:
you probably don't need scons, screen, htop, nano, vim. However it's nice to have them for debugging in case things break, it's up to you whether you want to keep it or not.

Reply:
Thank you. Will probably keep screen, and vim as this image is also so members can use it for dev if they want.

Reply:
Ah, if this has the possibility of being used for dev (I was thinking this one is just for CI), then all of these except scons might be useful. Scons is the old build system which I hope we are no longer using at all.